src/libs/transforms.ts

```diff
@@ -145,4 +145,17 @@ export function excludeHolidaySubscriptions(subcriptions: ZuoraSubscription[], n
   return subcriptions.filter(sub => {
     return !names.includes(sub.subscription_name);
   });
+}
+
+function subscriptionIsCorrect(subscription: ZuoraSubscription): boolean {
+  if(subscription.subscription_delivery_agent == ""){
+    return false;
+  }
+  return true;
+}
```

Comment:
do we need the if statement or could we just change the function to
`subscription.subscription_delivery_agent != ""`

Reply:
In this case, yeah, considering that there is only one condition that makes sense. I was about to add other conditions when we decided not to.

Reply:
https://github.com/guardian/national-delivery-fulfilment/pull/8